apps/chat/src/components/Marketplace/TabRenderer.tsx

```diff
@@ -293,51 +330,41 @@ export const TabRenderer = ({ screenState }: TabRendererProps) => {
       ) : (
         <>
           {selectedTab === MarketplaceTabs.MY_APPLICATIONS &&
-          suggestedResults.length ? (
-            <>
-              <div className="mb-8 flex items-center gap-1">
-                <Magnifier height={32} width={32} className="text-secondary" />
-                <span className="text-base">
-                  {t(
-                    'No results found in My workspace. Look at suggested results from DIAL Marketplace.',
-                  )}
+          isAllFiltersEmpty ? (
```

Comment:
it's hard to read such conditions. Maybe we can create some View component and return JSX depending on some conditions?